cord/client.py

```diff
@@ -384,6 +386,31 @@ def get_project(self):
         """
         return self._querier.basic_getter(Project)
 
+    def add_users(self, user_emails: List[str], user_role: ProjectUserRole):
```

Comment:
It's important for people to be able to quickly discern what value they're going to get back. Can you please add a type hint for the return value?